### DISCUSSION TITLE
google/protobuf/compiler/command_line_interface.h: present but cannot be compiled

### DISCUSSION THREAD

**Discussion by fiasthorton** [protobuf-c/protobuf-c#674] [Q&A]
git clone ptotobuf build and install successfully
then
git clone protobuf-c
configure failed with the following messages

checking pkg-config is at least version 0.9.0... yes
checking whether g++ supports C++11 features with -std=c++11... yes
checking for protobuf... yes
checking google/protobuf/compiler/command_line_interface.h usability... no
checking google/protobuf/compiler/command_line_interface.h presence... yes
configure: WARNING: google/protobuf/compiler/command_line_interface.h: present but cannot be compiled
configure: WARNING: google/protobuf/compiler/command_line_interface.h:     check for missing prerequisite headers?
configure: WARNING: google/protobuf/compiler/command_line_interface.h: see the Autoconf documentation
configure: WARNING: google/protobuf/compiler/command_line_interface.h:     section "Present But Cannot Be Compiled"
configure: WARNING: google/protobuf/compiler/command_line_interface.h: proceeding with the compiler's result
configure: WARNING:     ## -------------------------------------------------------------- ##
configure: WARNING:     ## Report this to https://github.com/protobuf-c/protobuf-c/issues ##
configure: WARNING:     ## -------------------------------------------------------------- ##
checking for google/protobuf/compiler/command_line_interface.h... no
configure: error: required protobuf header file not found

the config.log
[config.log](https://github.com/protobuf-c/protobuf-c/files/10370598/config.log)

**Comment by tbvdm** [comment]
It seems this is caused by protocolbuffers/protobuf@54346a414da72fe37ef3371cb3baddbd097cd501. Try building protobuf-c with `CXXFLAGS=-std=c++14` or higher.
